MKOB/kobactions.py

```diff
@@ -25,34 +25,28 @@
 """
 kobactions.py
 
-Defines actions for MKOB
+Handle actions for controls on main MKOB window
 """
 
 import tkinter.messagebox as mb
+import time
 from pykob import config, kob, internet, morse
 import kobconfig as kc
+import kobmain as km
 import kobstationlist as ks
-import time
 
 import pykob  # for version number
 print("PyKOB " + pykob.VERSION)
```

Comment:
Even though it's in the MKOB title bar - since you are printing this in the console it would be nice to print the MKOB version in the console as well.

Reply:
> Even though it's in the MKOB title bar - since you are printing this in the console it would be nice to print the MKOB version in the console as well.

No problem. I've made the change and it'll show up in the next pull request.